torch/optim/swa_utils.py

```diff
@@ -95,6 +95,8 @@ class AveragedModel(Module):
             an equally weighted average is used (default: None)
         use_buffers (bool): if ``True``, it will compute running averages for
             both the parameters and the buffers of the model. (default: ``False``)
+        should_deepcopy (bool): if ``True``, it will deepcopy the model. This is useful
+            to set ``False`` if model needs to be preprocessed prior, like wrapped in FSDP
```

Comment:
```suggestion
        should_deepcopy (bool): if ``True``, the constructor will deepcopy the model passed in to
        compute averaging. Otherwise, if ``False``, the constructor will compute averaging with the
        passed in model. Note that, in this case, the model passed in should NOT be the same as the
        original. This constructor is expecting that you have deepcopied the model yourself externally.
        This is useful if the model copy needs to be preprocessed prior, like wrapped in FSDP.
```

We should be quite explicit here since this is not the most intuitive